examples/simpleapp/mc_sc_workflow_example.md

```diff
@@ -66,14 +66,31 @@ Example of output is:
 Note: `withdrawalEpochLen` - expected to be in a range between 100 and 900 and is the same value as the one declared by `sc_create` command below.
 
 
-**Step 5: Generate data for certificate proof creation and verification**
+**Step 5: Generate Schnorr keypairs for certificate signers**
+
+As a next step we need schnorr keypairs, those keys will be used by certificate Signers
+
+For generating Schnorr key use command:
+`GenerateSchnorrKey {"seed":"my seed"}`
+
+Example output of newly created key pair is:
+```
+{
+    "schnorrPublicKey": "24be6e17b1b523a40c12ae2523633922ec405a41983268bae65717ce89202d3c00"
+    "schnorrSecret": "0400000020930c3927bbdf74fffd54860bf3a6192488493f116040381dbaa63f91a0e68c1324be6e17b1b523a40c12ae2523633922ec405a41983268bae65717ce89202d3c00",
+}
+```
+
+For retrieving other Schnorr keys repeat this command.
+
+**Step 6: Generate data for certificate proof creation and verification**
 
 Creation of Sidechains requires data for proving backward transfer operations, included in certificates. That data could be generated by next command:
 
-`generateCertProofInfo {"seed": "seed", "maxPks": 7, "threshold": 5, "verificationKeyPath": "/tmp/sidechainapp/cert_marlin_snark_vk", "provingKeyPath": "/tmp/sidechainapp/cert_marlin_snark_pk", "isCSWEnabled": true}`
+`generateCertProofInfo {"Pks": [pk1, pk2, ...], "threshold": 5, "verificationKeyPath": "/tmp/sidechainapp/cert_marlin_snark_vk", "provingKeyPath": "/tmp/sidechainapp/cert_marlin_snark_pk", "isCSWEnabled": true}`
```

Comment:
Let's rename **Pks** to **signersPublicKeys** to give more context to this argument.
And let's update the command example like:
`generateCertProofInfo {"signersPublicKeys": ["24be6e17b1b523a40c12ae2523633922ec405a41983268bae65717ce89202d3c00", ...], "threshold" ...`